datalad_neuroimaging/commands/create_study.py

```diff
@@ -0,0 +1,140 @@
+    ):
+        from datalad.support.constraints import EnsureKeyChoice
+        from datalad.distribution.dataset import Dataset
+        from datalad.distribution.install import Install
+        from datalad.distribution.siblings import Siblings
+
+        import os
+
+        for r in Create.__call__(
+                path, force, description,
+                dataset, no_annex, save,
+                annex_version,
+                annex_backend,
+                native_metadata_type,
+                shared_access, git_opts,
+                annex_opts, annex_init_opts,
+                text_no_annex,
+
+                # Don't implicitly depend on Create's defaults, but make them
+                # explicit instead:
+                result_xfm=None,
+                return_type='generator',
+                result_filter=EnsureKeyChoice('action', ('create',)) & \
+                            EnsureKeyChoice('status', ('ok', 'notneeded'))):
+            yield r
+
+            if r['type'] == 'dataset':
+                study_ds = Dataset(r['path'])
+
+                # TODO: get the actual container we are currently in
+                #       - requires to get info from outside (via ENV may be)
+                #       - but: we do we want to link to the _local_ container?
+                #       - may be always add psydata and/or github as a remote
+                # datalad run: option + config to use a paricular container
+                #              => set env to propagate container's dataset into
+                #                 the container itself
+                #              => SINGULARITYENV_ prefix
+                #
+                # datalad run "command" itself needs to set the env and call:
+                # THAT_ENV singularity exec configured/container  "command"
+
+                # TODO: Issue: If we clone from psydata, we get a message from
+                # datalad-install reading "access to dataset sibling
+                # "psydata-store" not auto-enabled, enable with:
+                # datalad siblings -d "/tmp/test/.datalad/environments/import-container"
+                # enable -s psydata-store
+                #
+                # => It's actually enabled. Issue comes from distribution/utils:_handle_possible_annex_dataset,
+                #    which looks whether that special remote is reported by name.
+                #    But it isn't, since it's also 'origin'. We would need to look for annexuuid instead.
+
+                # Note: This is a pointer to the container we are in, if it was
+                # passed by the caller (which could be datalad-run) via
+                # SINGULARITYENV_DATALAD_CONTAINER:
+
+                add_sibling_dicts = [{
+                    'action': 'add',
+                    'name': 'github-psyinf',
+                    'url': 'https://github.com/psychoinformatics-de/cbbs-imaging-container-import.git',
+                }]
```

Comment:
could/shouldn't those all be parts of the configurable items to be of general applicability?

Reply:
Yes, agree to all, hence description says: "Not ready yet. Just a translation of the original scripts."

Reply:
FTR: I also agree. ;-) Apart from that, this command should eventually go away in favor of a template to be called with `datalad-create` from core anyway.